patches/chromium/add_contentbrowserclient_ispluginsandboxdisabled_callback.patch

```diff
@@ -0,0 +1,109 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Milan Burda <milan.burda@gmail.com>
+Date: Sat, 22 Jun 2019 12:23:04 +0200
+Subject: Add ContentBrowserClient::IsPluginSandboxDisabled() callback
+
+Allows the embedder to decide whether the plugin should be sandboxed.
```

Comment:
Please upstream this patch & add a link to the chromium codereview :)

Reply:
I have never upstreamed stuff to Chromium so far, I need to try it for the first time. Any tips?

Reply:
Oops, I meant to respond to this ages ago. The best place to start is https://chromium.googlesource.com/chromium/src/+/master/docs/contributing.md

tldr: get a chromium checkout, make a branch, and run `git cl upload`.